### PR TITLE
bridge ver bump

### PR DESCRIPTION
**Pull request by butlerx**
- bump go-discord-irc version to fix broken webhook issue
- use correct hash for ssl

cc @ylmcc

### CODE REVIEW

**Comment by ylmcc** [comment]
Approved

File: services/ircd/inspircd/conf.nix [hunk 1]
```diff
@@ -386,7 +386,7 @@ in {
       certfile = "${config.security.acme.certs."irc.${tld}".directory}/fullchain.pem";
       keyfile = "${common.certsDir}/irc.${tld}/key.pem";
       dhfile = config.security.dhparams.params.ircd.path;
-      hash="sha256";
+      hash="sha1";
```
Comment (m1cr0man) [inline services/ircd/inspircd/conf.nix:389]: Why is this needed?

Comment (butlerx) [reply services/ircd/inspircd/conf.nix:389]: i only found the above broken when I put it into prod, ircd needs a testing env

**Review by m1cr0man — APPROVED** [review]
